docs/concepts/clustering.rst

```diff
@@ -138,48 +137,50 @@ the next update.
 Master Node Election
 --------------------
 
-The process of electing a node as the master node in a cluster is called Master
-Node Election. There must be only one master node per cluster at any single
-time. In a CrateDB cluster any node is eligible to be elected as a master node,
-although this could also be restricted to a subset of nodes if required. These
-nodes will then elect a single node as master to coordinate the cluster state
-across the cluster.
-
+In a CrateDB cluster there can only be one master node at any single time.
```

Comment:
I think that now, the explanation misses the point of the quorum, which is to avoid the `split-brain` situations.

Reply:
This is under the "Master Node Election" header, but a bit below in lines 151-153 I mention that the system is designed to prevent split-brain situations.

Reply:
Sorry, yes, please scratch that, I added this comment before reading the whole paragraph and forgot to delete it before finalizing the review.